lib/imageProcessor.js

```diff
@@ -153,11 +154,11 @@ ImageProcessor.prototype.handleAvatarUpload = function(file, destDir) {
 	// scale avatar
 	.then(function(oinfo) {
 		info.original = oinfo
-		return that.createScaledAvatar(normalizedPath)
+		return that.createScaled(normalizedPath, width, height, gravity)
```

Comment:
how about just `'Center'` or a `const DEFAULT_GRAVITY = 'Center'` somewhere in the beginning of the file instead of the unchanging argument?